r/tools/nixlibs.R

```diff
@@ -536,7 +536,7 @@ build_libarrow <- function(src_dir, dst_dir) {
   }
   cleanup(build_dir)
 
-  env_var_list <- c(
+  env_var_list <- list(
```

Comment:
This isn't technically necessary, but it was the easiest way to do what I wanted to in `is_feature_requested()`. Further, it's a bit strange that this thing called `X_X_list` is *not* a list.